public/class-h5p-plugin.php

```diff
@@ -98,7 +98,7 @@ private function __construct() {
     add_action('init', array('H5P_Plugin', 'check_for_updates'), 1);
 
     // Add menu options to admin bar.
-    add_action('admin_bar_menu', array($this, 'admin_bar'));
+    add_action('admin_bar_menu', array($this, 'admin_bar'), 999);
```

Comment:
A smaller number would definitely work. I can find plenty of documentation on the order of items across the admin bar, but ordering the nodes underneath an established menu item is less clear.

In my experiments, 70 still leaves the menu at the top, but 80 drops it all the way to the bottom. Initially I was attempting to drop the H5P menu under Post. When injecting within the established nodes didn't look like a viable option, I went with the bottom rather than the top. 

Numbers between 80-999 still leaves it higher than other plugins like Gravity Forms, in my install. 

![2017-05-30_08-52-57](https://cloud.githubusercontent.com/assets/480667/26584301/58d59ac0-4516-11e7-9215-36a1bb4b1210.png)